packages/select/src/token.ts

```diff
@@ -35,5 +35,10 @@ export const selectGroupKey: InjectionKey<SelectGroupContext> = Symbol('SelectGr
 export const selectKey: InjectionKey<SelectContext> = Symbol('Select')
 
 export const selectEvents = {
-  queryChange: 'elOptionGroupQueryChange',
+  elOptionGroup: {
```

Comment:
I would personally prefer using flat structure, say two names for different `queryChange`: queryChange and groupQueryChange.

Reply:
Fixed